classes/Environment.php

```diff
@@ -39,13 +44,25 @@ public static function testing(): self
         return new self(self::TYPE_TESTING);
     }
 
+    /**
+     * @throws \InvalidArgumentException
```

Comment:
Probably not quite appropriate to throw an `InvalidArgumentException` here, as the method itself doesn't accept any arguments.